node/lib/util/stitch_util.js

```diff
@@ -961,13 +962,31 @@ exports.stitch = co.wrap(function *(repoPath,
                                                   adjustPath);
         console.log("Found", Object.keys(fetches).length, "subs to fetch.");
         const subNames = Object.keys(fetches);
+        let subsRepo = repo;
+        const config = yield repo.config();
+        /*
+         * The stitch submodules repository is a separate repository
+         * to hold just the submodules that are being stitched (no
+         * meta repository commits).  It must be an alternate
+         * of this repository (the repository that stitching is
+         * being done in), so that we can access the objects
+         * that we fetch to it.  This lets us have a second alternate
+         * repository for just the meta repository commits.  And that
+         * saves a few seconds on meta repository fetches.
+         */
+        const subsRepoPath = yield ConfigUtil.getConfigString(
+            config,
+            "gitmeta.stitchSubmodulesRepository");
```

Comment:
The function is exported (public), so could you also mention this config string in the function doc?

Reply:
I don't think that makes sense?  It's an internal detail of the stitching process, so callers of the outer method probably don't need to be aware of it.  I added a doc instead.